packages/p2p/src/components/my-profile/payment-methods/add-payment-method/cancel-add-payment-method-modal.jsx

```diff
@@ -58,9 +60,8 @@ const CancelAddPaymentMethodModal = ({ is_floating }) => {
                     large
                     onClick={() => {
                         my_profile_store.setIsCancelAddPaymentMethodModalOpen(false);
-                        my_profile_store.setSelectedPaymentMethod('');
-                        my_profile_store.setSelectedPaymentMethodDisplayName('');
                         my_profile_store.hideAddPaymentMethodForm();
+                        my_profile_store.setIsCancelEditPaymentMethodModalOpen(false);
```

Comment:
Fixed a bug where when the user is shown the edit payment method form, the cancel edit payment method modal is open:
`Click Add New > Select a payment method > Click back arrow in header > Click Cancel > Edit a payment method`